stable/artifactory/templates/artifactory-statefulset.yaml

```diff
@@ -238,6 +241,9 @@ spec:
           mountPath: "/replicator_extra_conf/replicator.yaml"
           subPath: replicator.yaml
       {{- end }}
+        - name: installer-info
+          mountPath: "/tmp/info/installer-info.json"
```

Comment:
Let's try to mount to `/artifactory_extra_conf/info/installer-info.json`.
The entrypoint has `cp -rfv ${ARTIFACTORY_EXTRA_CONF}/* ${ARTIFACTORY_HOME}/etc`.